packages/wrangler/src/queues/cli/commands/consumer/remove.ts

```diff
@@ -0,0 +1,43 @@
+import { type Argv } from "yargs";
+import { logger } from "../../../../logger";
+import * as Client from "../../../client";
+import * as Config from "../../config";
+
+interface Args extends Config.Args {
+	["queue-name"]: string;
+	["script-name"]: string;
+	["environment"]: string | undefined;
+}
+
+export function Options(yargs: Argv): Argv<Args> {
```

Comment:
In general the convention with Typescript (that Wrangler tries to follow), is to use `camelCase` for variables/functions, and `PascalCase` for types/interfaces/classes. Would it be possible to match that for consistency with the rest of the codebase?

Reply:
Sure thing, cleaned this up.